### PR TITLE
Create fluentui/react-charting package to alias uifabric/charting

### LINKED ISSUE #24416: @fluentui/react-charting (re-exports @uifabric/charting)

### PR DESCRIPTION
**Pull request by micahgodbolt**
fixes #24416

This PR creates an alias package in v7 to allow v7 users to move to @fluentui/react-charting before making the upgrade to v8

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">7.0 Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/BaseButton-mount_base.html">1013</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/BaseButton-mount.html">1031</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Breadcrumb-mount_base.html">45751</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Breadcrumb-mount.html">45065</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Checkbox-mount_base.html">1862</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Checkbox-mount.html">1773</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/CheckboxBase-mount_base.html">1506</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/CheckboxBase-mount.html">1482</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/ChoiceGroup-mount_base.html">5480</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/ChoiceGroup-mount.html">5486</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/ComboBox-mount_base.html">964</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/ComboBox-mount.html">976</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/CommandBar-mount_base.html">8851</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/CommandBar-mount.html">8636</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/ContextualMenu-mount_base.html">15853</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/ContextualMenu-mount.html">16487</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/DefaultButton-mount_base.html">1258</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/DefaultButton-mount.html">1279</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/DetailsRow-mount_base.html">4255</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/DetailsRow-mount.html">4237</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/DetailsRowFast-mount_base.html">4245</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/DetailsRowFast-mount.html">4205</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/DetailsRowNoStyles-mount_base.html">4011</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/DetailsRowNoStyles-mount.html">3971</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Dialog-mount_base.html">2493</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Dialog-mount.html">2478</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/DocumentCardTitle-mount_base.html">2003</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/DocumentCardTitle-mount.html">1944</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Dropdown-mount_base.html">2837</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Dropdown-mount.html">2988</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/FocusTrapZone-mount_base.html">1890</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/FocusTrapZone-mount.html">1795</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/FocusZone-mount_base.html">1949</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/FocusZone-mount.html">1900</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/IconButton-mount_base.html">1974</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/IconButton-mount.html">1998</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Label-mount_base.html">362</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Label-mount.html">361</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Layer-mount_base.html">2168</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Layer-mount.html">2225</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Link-mount_base.html">511</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Link-mount.html">481</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/MenuButton-mount_base.html">1669</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/MenuButton-mount.html">1742</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/MessageBar-mount_base.html">2269</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/MessageBar-mount.html">2199</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Nav-mount_base.html">3612</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Nav-mount.html">3632</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/OverflowSet-mount_base.html">1527</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/OverflowSet-mount.html">1561</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Panel-mount_base.html">1598</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Panel-mount.html">1573</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Persona-mount_base.html">1088</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Persona-mount.html">1060</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Pivot-mount_base.html">1611</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Pivot-mount.html">1629</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/PrimaryButton-mount_base.html">1412</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/PrimaryButton-mount.html">1446</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Rating-mount_base.html">8855</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Rating-mount.html">8979</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/SearchBox-mount_base.html">1435</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/SearchBox-mount.html">1480</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Shimmer-mount_base.html">2928</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Shimmer-mount.html">2838</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Slider-mount_base.html">1661</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Slider-mount.html">1692</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/SpinButton-mount_base.html">5533</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/SpinButton-mount.html">5640</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Spinner-mount_base.html">450</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Spinner-mount.html">441</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/SplitButton-mount_base.html">3521</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/SplitButton-mount.html">3552</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Stack-mount_base.html">541</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Stack-mount.html">540</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/StackWithIntrinsicChildren-mount_base.html">1988</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/StackWithIntrinsicChildren-mount.html">2043</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/StackWithTextChildren-mount_base.html">5596</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/StackWithTextChildren-mount.html">5608</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/SwatchColorPicker-mount_base.html">11753</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/SwatchColorPicker-mount.html">11795</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/TagPicker-mount_base.html">3197</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/TagPicker-mount.html">3187</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/TeachingBubble-mount_base.html">54524</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/TeachingBubble-mount.html">54747</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Text-mount_base.html">450</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Text-mount.html">494</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/TextField-mount_base.html">1588</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/TextField-mount.html">1581</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Toggle-mount_base.html">923</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/Toggle-mount.html">923</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/button-mount_base.html">110</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24415/perf-test/button-mount.html">114</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: packages/react-charting/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# @uifabric/charting
```
Comment (dzearing) [inline packages/react-charting/README.md:1]:
```suggestion
# @fluentui/react-charting
```

File: packages/react-charting/package.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    "@types/d3-array": "1.2.1",
+    "@types/d3-axis": "1.0.10",
+    "@types/d3-format": "^1.3.1",
+    "@types/d3-scale": "2.0.0",
+    "@types/d3-selection": "1.4.1",
+    "@types/d3-shape": "^1.2.3",
+    "@types/d3-time-format": "^2.1.0",
+    "@uifabric/icons": "^7.7.5",
+    "@uifabric/set-version": "^7.0.24",
+    "@uifabric/charting": "4.21.10",
```
Comment (dzearing) [inline packages/react-charting/package.json:58]:
```suggestion
    "@uifabric/charting": "^4.21.10",
```

File: packages/react-charting/src/version.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+// Do not modify this file; it is generated as part of publish.
+// The checked in version is a placeholder only and will not be updated.
+import { setVersion } from '@uifabric/set-version';
+setVersion('@uifabric/charting', '0.0.0');
```
Comment (dzearing) [inline packages/react-charting/src/version.ts:4]:
```suggestion
setVersion('@fluentui/react-charting', '0.0.0');
```

File: packages/react-charting/CHANGELOG.json [hunk 4]
```diff
@@ -0,0 +1,7566 @@
+{
+  "name": "@uifabric/charting",
```
Comment (dzearing) [inline packages/react-charting/CHANGELOG.json:2]: delete this to reset changelog for this version.

Comment (dzearing) [inline packages/react-charting/CHANGELOG.json:2]:
```suggestion
  "name": "@fluentui/react-charting",
```

File: packages/react-charting/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,3230 @@
+# Change Log - @uifabric/charting
```
Comment (dzearing) [inline packages/react-charting/CHANGELOG.md:1]:
```suggestion
# Change Log - @fluentui/react-charting
```

File: packages/react-charting/README.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+**Charting components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+([formerly Office UI Fabric React](https://developer.microsoft.com/en-us/office/blogs/ui-fabric-is-evolving-into-fluent-ui/))
+
+The charting components here are ready to be used in production environment. However there are might be API revisions before final release.
+
+To import charting components:
+
+```js
+import { ComponentName } from '@uifabric/charting/lib/ComponentName';
```
Comment (dzearing) [inline packages/react-charting/README.md:11]:
```suggestion
import { ComponentName } from '@fluentui/react-charting/lib/ComponentName';
```

**Review by dzearing — APPROVED** [review]

**Pull request merged**